apps/frontend/src/store/snackbar.ts

```diff
@@ -20,25 +21,44 @@ export interface ISnackbarState {
   name: 'SnackbarModule'
 })
 export class Snackbar extends VuexModule {
+  customText: string | undefined = 'owo';
```

Comment:
Not sure I understand the point of the customText. It seems to only be set and never used.

Reply:
Oh sorry, that was for debugging and I never removed it.

Reply:
This was intended to add a button with a custom url to the releases page, customText being "View", and url being the link to the releases page